lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -28,17 +24,24 @@ def tensor_from_series(series, device, n_dims, pad_value, max_len=None, normaliz
             series[i].append(pad_value)
         series[i] = series[i][:max_len]
 
-    # dimension padding
-    for _ in range(max(0, n_dims - len(series))):
-        series.append([pad_value] * max_len)
-
-    # normalize and transpose
     if normalizer:
-        tensor = torch.Tensor([normalizer.encode(s) for s in series][0])
+        if isinstance(normalizer, MinMaxNormalizer):
+            tensor = torch.Tensor([normalizer.encode(s) for s in series]).transpose(0, 1)
+        else:
+            tensor = torch.Tensor([normalizer.encode(s) for s in series][0])
+
+        if len(tensor.shape) > 2:
+            tensor = tensor.squeeze(2)
```

Comment:
When can this be the case ? Is it not only when a normalizer other then MinMax is used (i.e. the date one) ? If so, just make it part of that `else`.

Reply:
Never, it's a leftover that I forgot about, good catch

Reply:
Scratch that, it is used and only for the MinMax, point taken.